full-moon/src/ast/mod.rs

```diff
@@ -206,6 +213,16 @@ pub enum Field {
         value: Expression,
     },
 
+    /// A set constructor field, such as .a inside { .a } which is equivalent to { a = true }
+    #[display("{dot}{name}")]
+    #[cfg(feature = "cfxlua")]
+    SetConstructorField {
```

Comment:
We are already in Field, we don't need `Field` at the end of the name
```suggestion
    SetConstructor {
```